deploy/helm/scf/assets/operations/instance_groups/diego-cell.yaml

```diff
@@ -42,30 +41,88 @@
 - type: replace
   path: /instance_groups/name=diego-cell/jobs/name=cfdot/properties/locket?/hostname
   value: ((deployment-name))-diego-api
-
-# Add quarks properties.
 - type: replace
-  path: /instance_groups/name=diego-cell/jobs/name=garden/properties/quarks?
-  value:
-    run:
-      memory: 4096
-      virtual-cpus: 2
-      # healthcheck:
-      #   garden:
-      #     readiness: &probe
-      #       exec:
-      #         command: ['sh', '-c', 'ss -nlt | grep "LISTEN.*:17019"']
-      #     liveness: *probe
+  path: /instance_groups/name=diego-cell/jobs/name=rep/properties/set_kernel_parameters?
+  value: false
 
 # Temporarily remove garden network_plugin.
 - type: remove
   path: /instance_groups/name=diego-cell/jobs/name=garden/properties/garden/network_plugin
 - type: remove
   path: /instance_groups/name=diego-cell/jobs/name=garden/properties/garden/network_plugin_extra_args
 
+# Add bosh_containerization properties for the rep.
+- type: replace
+  path: /instance_groups/name=diego-cell/jobs/name=rep/properties/bosh_containerization?
+  value:
+    ports:
+    - name: rep
+      protocol: TCP
+      internal: 1800
+    - name: rep-tls
+      protocol: TCP
+      internal: 1801
+
 - type: replace
   path: /instance_groups/name=diego-cell/jobs/name=cflinuxfs3-rootfs-setup/properties/quarks?/bpm/processes
   value: []
 - type: replace
   path: /instance_groups/name=diego-cell/jobs/name=cfdot/properties/quarks?/bpm/processes
   value: []
+
+- type: replace
+  path: /instance_groups/name=diego-cell/jobs/name=rep/properties?/diego?/rep?/locket?/api_location?
+  value: ((deployment-name))-diego-api:8891
+
+- type: replace
+  path: /instance_groups/name=diego-cell/jobs/name=rep/properties?/diego?/rep?/bbs?/api_location?
+  value: ((deployment-name))-diego-api:8889
+
+# TODO: the namespace here has to be determined automatically - helm can help here?
```

Comment:
I'd create an implicit variable that is fed from values.yaml. We can't use Helm's `.Release.Namespace` with `helm template`, which is our current approach to avoid any Tiller magic.

Reply:
If we use a combination of `hostname` and `subdomain` with cf-operator, we can solve this TODO and the TODO below. I'm exploring this on the cf-operator side now.